iree/compiler/Dialect/HAL/Conversion/HALToVM/ConvertAllocatorOps.cpp

```diff
@@ -45,8 +45,19 @@ class AllocatorAllocateConstOpConversion
     auto constName = (parentFuncOp.getName() + "_const_" +
                       std::to_string(allocateUniqueId(parentFuncOp)))
                          .str();
+
+    // VM supports a minimum of 8-bit elements. Round 1-bit bools to 8-bits.
```

Comment:
```suggestion
    // VM supports a minimum of 8-bit elements. Expand 1-bit bools to 8-bits.
```

nit: "round 1-bit ... to 8-bits" could be more clear in what this is doing. How about "Expand 1-bit..." or "Cast 1-bit..."?

Reply:
Obsolete